legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -187,6 +187,17 @@ class FilingTitles(str, Enum):
             {'types': 'BC,BEN,CC,ULC', 'outputs': ['letterOfConsent', ]},
         ]
     },
+    'continuationOut': {
+        'name': 'continuationOut',
+        'title': 'Continuation Out',
+        'displayName': 'Instrument of Continuation Out',
```

Comment:
I don't know if this is what we want to display.  I think it should be "Continuation Out"?

![image](https://github.com/bcgov/lear/assets/6685223/8614a864-ac31-4c67-9ee2-7375fb62d014)